delta-api/src/main/java/io/cdap/delta/api/DDLOperation.java

```diff
@@ -36,12 +36,17 @@
     RENAME_TABLE
   }
 
+  private final String schema;
```

Comment:
can we move database name to this class too? from DMLEvent and DDLEvent? So that table identifier - databasename, schema and table name at the same place?

Reply:
done:) PTAL :)